### PR TITLE
Add logback logging

### PR DESCRIPTION
**Pull request by LukasGasior1**
File + stdout in app, stdout only in tests

### CODE REVIEW

File: src/it/scala/io/iohk/ethereum/mpt/MerklePatriciaTreeIntegrationSuite.scala [hunk 1]
```diff
@@ -4,22 +4,24 @@ import java.io.File
...
+import org.slf4j.LoggerFactory
 import org.spongycastle.util.encoders.Hex
 
 import scala.util.Random
 
 class MerklePatriciaTreeIntegrationSuite extends FunSuite
   with PropertyChecks
   with ObjectGenerators {
+
+  val log = LoggerFactory.getLogger(this.getClass)
```
Comment (adamsmo) [inline src/it/scala/io/iohk/ethereum/mpt/MerklePatriciaTreeIntegrationSuite.scala]: Don't Repeat Yourself

Comment (LukasGasior1) [inline src/it/scala/io/iohk/ethereum/mpt/MerklePatriciaTreeIntegrationSuite.scala]: Done

**Comment by LukasGasior1** [comment]
@etc-builder I pushed something similar just before your comment :)

**Comment by LukasGasior1** [comment]
@mcsherrylabs Does it make sense? In most cases we want the actual logging class name to show in log, not the Logger

**Comment by whysoserious** [comment]
>  In most cases we want the actual logging class name to show in log, not the Logger 

I'd use it logger directly as well.

**Comment by mcsherrylabs** [comment]
I think it's worth adding. The 'getLogger' allows the conscientious to pass a category. Mixing in a trait might not always be appropriate/possible. 

object Logger extends Logging {

  def getLogger(category: String): Logger = {
    LoggerFactory.getLogger(category)
  }

}

**Comment by LukasGasior1** [comment]
I don't see much difference in:
`LoggerFactory.getLogger(getClass)` vs our `Logger.getLogger(getClass)` but ok, I'll add it

**Comment by LukasGasior1** [comment]
@mcsherrylabs done

**Pull request merged**